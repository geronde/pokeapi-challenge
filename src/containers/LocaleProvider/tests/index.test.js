import React from 'react';
import {render} from '@testing-library/react'
import LocaleContextProvider from '..';

describe('LocalProvider',()=>{
    let props;
    beforeEach(() => {
        props = {
            children: <div>hello</div>
        }
    });

    it('should render LocaleContextProvider correctly', () => {
        const {asFragment} = render(<LocaleContextProvider {...props} />)

        expect(asFragment()).toMatchSnapshot();
    });
})