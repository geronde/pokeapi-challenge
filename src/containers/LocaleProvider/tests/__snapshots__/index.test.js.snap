// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocalProvider should render LocaleContextProvider correctly 1`] = `
<DocumentFragment>
  <div>
    hello
  </div>
</DocumentFragment>
`;
